modules/labs-react/toast/stories/toast.stories.mdx

```diff
@@ -0,0 +1,165 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Toast} from '@workday/canvas-kit-labs-react/toast';
+import {Basic} from './examples/Basic';
+import {ToastAlert} from './examples/ToastAlert';
+import {WithCloseButton} from './examples/WithCloseButton';
+import {ToastDialog} from './examples/ToastDialog';
+import {WithActionLinkAndCloseIcon} from './examples/WithActionLinkAndCloseIcon';
+import {WithPopper} from './examples/WithPopper';
+import {RTL} from './examples/RTL';
+
+<Meta title="Labs/Toast/React" component={Toast} />
+
+# Canvas Kit Toast
+
+`Toast` is a [compound component](/getting-started/for-developers/resources/compound-components/)
+that contains updates or messages about the status of an application's process.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-labs-react
+```
+
+## Usage
+
+### Basic Example
+
+`Toast` includes a container `Toast` component and the following subcomponents: `Toast.Body`,
+`Toast.Close`, `Toast.Icon`, `Toast.Message`, and `Toast.Link`.
+
+`Toast` supports different modes through the `mode` prop: `status`, `alert`, and `dialog`. Each mode
+conveys a different purpose of the message and assigns the necessary ARIA attributes to support that
+purpose and provide an accessible experience.
+
+By default, `mode` is set to `status`, which indicates the message contains advisory information
+such as a successful action.
+
+<ExampleCodeBlock code={Basic} />
+
+A `status` `Toast` will wrap the message within a `polite` ARIA live region with a `role` of
+`status`.
+
+Here's a more complete example with a button which triggers a dismissable `Toast`. The `Toast` is
+positioned using `Popper`.
+
+<ExampleCodeBlock code={WithPopper} />
+
+### Alert
+
+Set the `mode` prop to `alert` to convey urgent and important information such as an error.
+
+<ExampleCodeBlock code={ToastAlert} />
+
+An `alert` `Toast` will wrap the message within an `assertive` ARIA live region with a `role` of
+`alert`.
+
+### Dialog
+
+Set the `mode` prop to `dialog` to convey the presence of a follow-up action.
+
+<ExampleCodeBlock code={ToastDialog} />
+
+The `Toast` will be given a role of `dialog` along with an `aria-label` to indicate that it's a
+notification.
```

Comment:
Based on my talks with accessibility, role`alert` will read out the message in the toast and yes, both status and alert would go in a live region area! The dialog needs the describedby so that it reads the text of the toast that is associated with the toast. Here's what the accessibility tree looks like for dialog.

![Screen Shot 2022-08-29 at 8 23 09 AM](https://user-images.githubusercontent.com/7966550/187223849-650523aa-ce90-4c31-a607-87b468a4efbb.png)